lib/ansible/plugins/lookup/mongodb.py

```diff
@@ -62,7 +64,7 @@
             type: integer
         sort:
             description:
-                - Sorting rules. Please notice the constats are replaced by strings.
+                - Sorting rules. Please use the strings "ASCENDING" and "DESCENDING" to set the order. Check the example for more information.
```

Comment:
Please split up the description items. The first item should be an explanation of the description.
```suggestion
                - Sorting rules.
                - Please use the strings C(ASCENDING) and C(DESCENDING) to set the order.information.
```